### ISSUE TITLE
Request for automatically change dy to negative

### ISSUE THREAD

**Issue by Manangka** [Deltares/imod-python#93]
In GitLab by @JannekePouwels on Sep 26, 2019, 12:49

When writing a netcdf to an idf file, I get the error: ValueError: dy must be negative. I think it is better to change y-coordinates inside the imod.idf.write function, because it is a really technical error and therefore the error is confusing and hard to debug for imod-python users.

**Comment by Manangka** [comment]
In GitLab by @Huite on Feb 24, 2020, 11:23

This should be fixed: 0107256299bf89ee75a7d4d8ef296c9f7b9ef4a4

My current solution is:
```python
flip = slice(None, None, -1)
if not da.indexes["x"].is_monotonic_increasing:
    da = da.isel(x=flip)
```
Since selection is lazy, I would expect this not to create any copies. But we should check.

**Comment by Manangka** [comment]
In GitLab by @Huite on Feb 24, 2020, 11:23

closed
